File: Kattis-Problems/Python-Solutions/Pot.py
a = int(input())
arr = [input() for i in range(a)]
sum = 0

for k in range(a):
    first = arr[k]
    second = arr[k]
    first = first[0:len(first)-1]
    second = second[len(second)-1:len(second)]
    sum += pow(int(first), int(second))

print(sum)

File: Kattis-Problems/CPP-Solutions/Pot.cpp
#include <iostream>
#include <cmath>

using std::cout;
using std::cin;
using std::endl;

int main(){
    int a, b, sum = 0;
    cin >> a;
    for(int i = 0; i < a; ++i){
        cin >> b;
        sum += pow( (b/10) , (b%10));
    }
    cout << sum;
}
